src/components/general/EntryCommentModal/CommentForm/index.tsx

```diff
@@ -98,8 +96,8 @@ function CommentForm(props: Props) {
             }));
         }
         setPristine(true);
-        postComment();
-    }, [setPristine, postComment]);
+        postComment(faramValues);
+    }, [setPristine, postComment, faramValues]);
```

Comment:
This function should not call setFaramValues.
And in postCommit should use newFaramValues directly